extensions/ql-vscode/src/skeleton-query-wizard.ts

```diff
@@ -201,9 +201,17 @@ export class SkeletonQueryWizard {
     });
 
     const githubRepoNwo = this.QUERY_LANGUAGE_TO_DATABASE_REPO[this.language];
+    const chosenRepo = await databaseFetcher.askForGitHubRepo(
+      this.progress,
+      githubRepoNwo,
+    );
+
+    if (!chosenRepo) {
+      throw new Error("No GitHub repository provided");
```

Comment:
Should this be a `UserCancellationException` so this doesn't get send to our telemetry?